src/main/webapp/app/entities/quiz/quiz-exercise.model.ts

```diff
@@ -23,6 +23,7 @@ export class QuizExercise extends Exercise {
     public adjustedReleaseDate: Moment;
     public ended: boolean;
     public started: boolean;
+    public statusAsNumber: number;
```

Comment:
please don't add arbitrary elements here. At least document the meaning!

Reply:
I changed quizStatus to an Enum. The ordering can now simply be determined via the associated number for each number field.

Reply:
addressed via ea8ea787b8d8cf18d89270f9221a8815b15e0700